### ISSUE TITLE
Sharpening stones affecting hunter shots

### ISSUE THREAD

**Issue by Kiszh** [SunwellWoW/Sunwell-TBC-Bugtracker#4359]
Description: Hunters can apply sharpening stones to their melee weapons in order to increase the damage they do with shots. Adding 12 average damage to every shot. This correlates to 56 attack power with a 3.0 speed bow (12/3*14). This also means that dualwielding can give hunters an unintended equiviliant of 112+ attack power. 

How it works: Applying sharpening stones to melee weapons increase the damage of ranged attacks. 

How it should work: The sharpening stones should only add damage to melee attacks, same as scopes don't increase the damage of your melee swings.

Source (you should point out proofs of your report, please give us some source): https://wow.gamepedia.com/Coarse_Sharpening_Stone

Coarse Sharpening Stones are created by Blacksmiths and _will sharpen a sword, axe, dagger, or polearm to perform more damage._ 

An Adamantite Weightstone is a temporary enhancement for blunt weapons, like a mace or a staff. It improves _the weapon's damage_

https://wow.gamepedia.com/Adamantite_Weightstone

The tooltip ingame also states "increase blunt" or "increase sharp" weapon damage, and I believe ranged attacks are neither, but are classed as ranged.

Edit: I've anecdotally been told it affects pet damage aswell - atleast Wind Serpent breath

There is no proof outrighting stating that the sharpening stones should not affect ranged damage. There, however, is no mention of hunters using sharpening stones in any guides I've seen. If they did in fact function for hunters they are so strong, that they surely would've been mentioned somewhere.

**Comment by Highdps** [comment]
By KraxisSingular (1,076 – 1·10) on 2008/08/12 (Patch 2.4.3) |   |  
If you are a Hunter and use a Fist weapon, then this is the best single DPS buff you can get. Both the damage and the critrating applies to ranged weapons unlike the Sharpening Stones.

**Comment by Kiszh** [comment]
Are there any other proof out there aside from a (I assume wowhead) comment? There is also an old wowhead comment claiming that rod of the sun king has 27% proc chance with no ICD. 

That being said I still find it highly odd that this appears nowhere on the EJ guides or to my knowledge in the hunter spreadsheet. 

What I did find in wowhead comments was dissent among commentors on whether or not the damage buff applies to ranged attacks.

**Comment by Highdps** [comment]
Proof are experience 
Every hunter in late tbc (sunwell) was going for combo fist weapon (trash off-hand or KJ main hand) to be able to use Adamantite stones and Dagger for sunwell oil.
It should be point out somewhere in old EJ threads, but here all workings correct and it's works as intented.
Also Wowhead comment about 27% proc chance kael mace can be true, depending on patch/ptr server
We have 2.4.3 mechanics and we should get as close as possible to that.

**Comment by Kiszh** [comment]
27% proc chance on the mace, which is not the subject of the report, is in no way reliable and completely broken. The guy was simply wrong. 

![billede](https://user-images.githubusercontent.com/5422308/53228640-16579e00-3682-11e9-9479-e81b8c55cf88.png)

This is a screenshot from the hunterspreadsheet, in the SWP patch, with a fist equipped and there is no option for a weightstone, which is highly suspicious if that was indeed the prefered and well known way of raiding. 

I'm still ctrl + F'ing my way through EJ threads.

**Comment by Kiszh** [comment]
http://web.archive.org/web/20080929133925/http://elitistjerks.com/f31/t16943-hunter_lounge_-_theorycraft_within/p14/

No result

http://web.archive.org/web/20080502162354/http://elitistjerks.com/f31/t15179-hunter_help_me_please/p115/

Nothing in this thread either Only looked through posts from 08, which means SWP came out during.

http://web.archive.org/web/20080714071541/http://elitistjerks.com/f31/t12346-hunter_survival_raiding_2_1_a/p96/

Nothing here either

http://web.archive.org/web/20080405204218/http://elitistjerks.com/f31/t11420-hunter_beast_mastery_bible/p131/

No mention in this either all the way up to page 134 

The only hit I got on "adam" through all the threads was on "Adamantite arrowmaker". It's not even being talked about in any of these threads.

**Comment by Kiszh** [comment]
Finally found proof

Cheeky's notes dated 9/27/07
- Removed Sharpening Stones from weapon buffs.

09/25/07, 8:11 PM 
Quick FYI, sharpening stones are getting changed to not affect range.

https://web.archive.org/web/20081110194841/http://elitistjerks.com/f31/t11420-hunter_beast_mastery_bible/p93/

**Comment by Highdps** [comment]
It's Sharpening Stone, on Fist/maces hunter's using Adamantite Weightstone

**Comment by MiekZer** [comment]
Adamantite Sharpening Stone should only increase ranged damage and not critical hit since it's stated as melee critical hit.
Adamantite Weightstone should increase both since it's not stated as melee critical hit but just critical hit.

It's working as intended and was also working the same on other quality script pserv (warmane for example it's working like that)

Edit 1 : some comments on wowhead even stating this about adamantite weightstone in 3.0.3 can make us assume it was working like that during TBC and start of WOTLK at least (remember in wotlk there is no more spell crit or melee crit just crit). So the following comment and others on wowhead can confirm this was not scripted to only add crit and damage on the "melee" sheet stats of your character but on every sheets ("ranged", and "spell" during wotlk due to critical rating being same for physical and caster)
"Wasol on 2008/12/14 (Patch 3.0.3)
This is currently the best (and only working) weaponbuff for casters using staff/mace atm, sience they benefit from the 14 critrating."

**Comment by Kiszh** [comment]
At both of the new comments. Sharpening stone is the general term used about both the adamantite weightstone and adamantite sharpeningstone. I also never really spoke about the crit but the +damage portion of the sharpening stones, which according to elitist jerks got removed. So no "sharpening stones" should not give +12 damage on ranged attacks.

It's still rather suspect that adamantite sharpening stones were not mentioned anywhere in 4 different hunter EJ threads. Yes I've CTRL + F'd with "Adam" through hundreds of pages. The ONLY hit was "Adamantite Arrowmaker". It's also not present in the hunter spreadsheet as shown above.

Wowhead comments are really not proof of anything, and as you can see from the quoted EJ post above, it says that sharpening stones have been changed to no longer affect ranged attacks. 

It's not mentioned in any guide anywhere, which is odd for how good it is, that no one thought to mention that you should probably use those. Instead mana oils are mentioned. 

Funny that you should mention Warmane. I actually used their bugtracker to find the source for my proof, because it was also a bug there. Whether it was ever fixed or not I don't know. 

If you are taking wowhead comments as a viable source, you probably shouldn't ignore the posts also saying it does not work for hunters. Such as:

NobodyYouKnow on 2007/08/23 (Patch 2.1.3)
Unfortunately, this does only effects melee attacks. This does nothing at all for ranged attacks. Well, back to using lesser mana oil for me hunter!


  | Alegis on 2007/07/19  (Patch 2.1.3)
As of 2.2 these will no longer apply to ranged crit.
-- | --

The wowhead posts are simply contradicting eachother. The fact that hunters are not recommended to use these anywhere else is highly suspect. I'll gladly stand corrected, but it needs more than a wowhead post or two with no backing or ethos in terms of game mechanics. Otherwise we need to have a talk about ironfoe and how it's the most overpowered weapon in the game and how I should go farm it, because according to a wowhead comment it copies the attack it procs off x2.

**Comment by MiekZer** [comment]
You are very hypocrite.
Adamantite Weightstone and Adamantite Sharpening stone are 2 differents stones.
Every wowhead comments about Adamantite Weightstone states it as the best 2.4.3 weapon buff for hunter and states it as the best 3.0.3 buff for boosting crit for casters.

The wowhead comments you are refering to is a comment about Adamantite Sharpening Stone, which I recall is stated as MELEE CRIT. So yeah I assume it's a legit comment saying that as of 2.2 the crit chance will no longer increase ranged crit chance due to it's MELEE CRIT STATEMENT, we can then understand that it was probably bugged before 2.2 to also increase ranged crit chance. 

And by the way, your comment is ONLY stating ranged crit meaning the damage portion is still working after 2.2 for Adamantite Sharpening Stone on RANGED HITS.

EDIT1 : The same comment you quoted but one line after (About Adamantite Sharpening Stone) : 
"Killerrabbit le 2008-05-14 (Mise à jour 2.4.1)
Actually, it is exactly the opposite. Ranged attacks do not get the crit bonus, but they do get the +damage."             =========> 2.4.1 and he's saying it's working exactly the same as it's actually working on this server.


To conclude (for hunter) : 
every wowhead comments about Adamantite Weighstone say it's giving +damage and +crit.
every wowhead comments about Adamantite Sharpening Stone say it's only giving +damage, and some comments says it's not working because they were probably just watching their critical chance stat at this moment and concluded it didn't work at all when it's working for +damage.

about EJ:  their guides also say hunter must play with a macro and CoH shouldn't use blue dragon card and so on and so on...
And the guides you posted about sunwell optimization they are also not talking about righteous weapon coating (I let you google it)

**Comment by Kiszh** [comment]
Mind you that I'm not the one who dragged wowhead comments into this. All I'm asking is a higher standard of proof, because right now I have to prove a negative. 

You're asking me to prove that they didn't work, by finding proof that they didn't. That is nearly, if not completely, impossible. Instead we need proof that they did in fact work, which there should be plenty of archived evidence of. I'm finding none aside from a couple of wowhead comments, which is a low standard of proof.

There is, however,  proof that they did work once upon a time, but also proof that they at some point stopped working - from a source that is way more trustworthy than a few wowhead comments. That time going by the dates was patch 2.2.

We know for a fact, that until patch 2.2 hunters used sharpening stones, that is evident by the EJ post and the changelog to the spreadsheet. If adamantite weightstones still worked for hunters, why would that not have been left in the spreadsheet, and only the sharpening stones be removed?

Now for a second point I'll try to take this very slowly to avoid misunderstandings.

Currently the item called **_"Adamantite sharpening stone"_** is affecting the damage done by ranged attacks. In 2 different posts in september 2007 it was mentioned that "Sharpening stones" - which could only be those or could possibly also include weight stones - no longer affect ranged attacks. 

Common sense says that it applies to BOTH types of stones, because **both were removed** from the spreadsheet.

At the moment **"adamantite SHARPENING stone**" is affecting ranged attacks. Can we agree this should not be the case by the proof presented by the EJ posts?

You're right that the spreadsheet nor what I read mentioned the weapon coating, which is obviously bad. But please note that the tooltip on that item _specifically_ states that i affects ranged attacks. The _"Adamantite sharpening stone"_ and _"Adamantite weightstone"_ say they affect "**Blunt weapon damage**" and "**Sharp weapon damage**" there is nothing about this affecting ranged weapon damage. By your line of logic flametongue totem should affect ranged attacks aswell.

**Comment by MiekZer** [comment]
One guy call it on EJ without any proof and patchnote when he was maybe talking about the critical buff part of the sharpening stone (which was according to the 2.2 wowhead comment probably patched due to a bug) and every stones should no longer affect ranged attack when you have plenty of comments 2.4.3+ saying it's affecting ranged attacks, even spells?
Ok.

Some comments about other types of stones on Wowhead:
(Dense Sharpening Stone):
By Deltros (431 – 8·5) on 2008/10/04 (Patch 3.0.1) (first patch of wotlk and still working)
Interesting side feature for hunters especially
I'm not sure why, but for some reason this actually pumps up your RANGED damage as well. So far I've tested rough and dense, and they both have resulted in my Ranged damage getting a buff.
So...from what I can tell, it seems that it benefits hunters to keep a supply of sharpening stones on hand.

"For new hunters considering Blacksmithing (or just have access to this item and its kind in general), this item actually seems to buff ranged damage as well as melee. This confuses me a bit, honestly, HOW it works, but somehow the attack bonus from the sharpening stones goes onto your ranged weapon as well from one use on your melee weapon."

Stones were changed in 3.3.0:
Deltros on 2010/01/19 (Patch 3.3.0)
Update- as of this last patch or so, the sharpening stones that did increase Ranged Attack Power now ONLY affect the weapon that has been sharpened. ;-; Blast, Blizz fixed it.

**Comment by Kiszh** [comment]
I'll make it pretty simple.

Do you consider bows blunt weapons or sharp weapons? Because it literally says on the item that they increase blunt or sharp weapon damage. 

Once again you're using wowhead comments, and this time from Wotlk. 

The reason I am asking for more than wowhead comments is because comments on there are not reputable - in fact they are wrong a lot. Elitist jerks, however, was and is known to be a place you can trust. That being said they are not gods and obviously they are subject to being wrong. But the complete lack of mention in their guides or discussions is a very clear indicator of this being a bug. Otherwise ask yourself why no hunter in these threads never asked "Weight/sharpeningstone vs mana oil".

______________________________________________________________
Now I personally went through these threads CTRL + Fing "Adam" and I found NO mention of it at all. 

 http://web.archive.org/web/20080915224636/http://elitistjerks.com/f31/t16943-hunter_lounge_-_theorycraft_within/
    http://web.archive.org/web/20080912152455/http://elitistjerks.com/f31/t15179-hunter_help_me_please/
 http://web.archive.org/web/20080912004752/http://elitistjerks.com/f31/t12346-hunter_survival_raiding_2_1_a/
    http://web.archive.org/web/20080913121430/http://elitistjerks.com/f31/t9816-hunter_spreadsheets_--_development/
   http://web.archive.org/web/20080924170342/http://elitistjerks.com/f31/t11420-hunter_beast_mastery_bible/

There is no mention of it anywhere outside of the little note on the change in 2.2. 

When I went through the warrior threads I did find some discussion about it. I found some discussion about it, because it's a consumable that affects warriors, naturally warriors talk about it.

What I did not find was discussions about stuff that doesn't affect warriors, such as scopes. There is also no mention about ranged attacks being affected for warriors. It seems odd considering the amount of time they spent testing to the point where they even showed it affects bloodthirst, which they had no idea about.

**Comment by MiekZer** [comment]
EJ is not a bible.
You can also ask yourself why no one ever talked about the fucking righteous weapon coating when they're talking about sunwell plateau optimization.
Read 95% wowcomments stating it's working as intended and close the issue bro

**Comment by Kiszh** [comment]
I stand corrected.

I finally managed to get access to the full thread instead of having some pages unavailable, and it seems that both sharpening stones and weightstones started granting their +damage modifer on range attacks somewhere around patch 2.4.2 (May 2008), while between patch 2.2 (september 2007) and 2.4.2 they did not. 

I would, however, like to point out that I sourced this from EJ and not Wowhead, as Wowhead is still not to be considered definitive proof of anything. 

For anyone interested the source is: http://web.archive.org/web/20080529002505/http://elitistjerks.com/f31/t15179-hunter_help_me_please/p123/

around page 123 and onwards sharpening stones make an appearance in hunter consumables once again. Untill then only mana oils are mentioned, while on page 18 it's mentioned that sharpening stones no longer affect ranged attacks. 


I do have to ask, however, whether it's normal that the hunter pet gain the +damage modifier aswell, as that might be a bug. I see that benefit mentioned nowhere, not on EJ nor on wowhead. 

From what I gather hunter pets scale like this:
Pet's armor bonus = 0.35 hunter's armor bonus
Pet's stamina bonus = 0.30 Hunter's stamina bonus
Pet's atack power = 0.22 Hunter's attack Power

Nothing about pets scaling with +damage modifiers. I however am not 100% sure this is occuring, if any hunter could test it.
